src/Initializer/PointMapper.cpp

```diff
@@ -136,7 +139,7 @@ void findMeshIds(const Eigen::Vector3d* points,
         for (unsigned dim = 0; dim < 4; ++dim) {
           resultFace += planeEquations[dim][face] * points1[point][dim];
         }
-        notInside += (resultFace > 0.0) ? 1 : 0;
+        notInside += (resultFace > tolerance) ? 1 : 0;
```

Comment:
I agree, 
So if the fault receiver is on the fault, at the intersection between 2 faults, we get 2 fault receivers of same location, but with slightly different output, right?
Also if the receiver is outside of the fault, at the boundary between 2 tetrahedra, then we get 2 receivers, of different locations (if the 2 fault elements have different cells).

Reply:
Not having duplicated receiver is good.
probably the correct way would be to use the solution of the Riemann problem for receivers exactly at the interface between 2 cells (instead of arbitrarily evaluating in one cell).
(but let's forget about that :-p)